src/experimental.rs

```diff
@@ -0,0 +1,18 @@
+//! Experimental methods.
+
+use super::*;
+
+mod native {
+    extern "C" {
+        pub fn experimental_isAccountEmpty(addressOffset: *const u32) -> u32;
```

Comment:
Should this be under `experimental` namespace in the EEI? I think this particular one is under `ethereum`, just this whole file would collect the non-final methods.